examples/paddle/conda.yaml

```diff
@@ -0,0 +1,13 @@
+name: paddle-example
+channels:
+- defaults
+- conda-forge
+dependencies:
+- python=3.6.13
+- pip
+- pip:
+  - mlflow
+  - paddlepaddle==2.1.0
+  - cloudpickle==1.6.0
+
+    
```

Comment:
Can we remove the blank lines here?

Reply:
Sorry for carelessness...

Reply:
Please don't be sorry, I also make this mistake a lot :)